cmd/talosctl/cmd/talos/time.go

```diff
@@ -54,7 +54,7 @@ var timeCmd = &cobra.Command{
 			}
 
 			w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
-			fmt.Fprintln(w, "NODE\tNTP-SERVER\tLOCAL-TIME\tREMOTE-TIME")
+			fmt.Fprintln(w, "NODE\tNTP-SERVER\tMACHINE-TIME\tNTP-SERVER-TIME")
```

Comment:
I hate to be the dissenter, but shouldn't this be `NODE-TIME` instead of `MACHINE-TIME`?  They are generally called nodes in talosctl.

Reply:
NODE-TIME it is. fixed in amended commit.